ament_uncrustify/ament_uncrustify/main.py

```diff
@@ -282,8 +202,13 @@ def find_executable(file_name, additional_paths=None):
     return shutil.which(file_name, path=path)
 
 
-def get_files(paths, extensions, excludes=[]):
-    files = []
+def get_files(paths, extension_types, excludes=[]):
+    extensions_with_dot_to_language = {
+        (language, extension)
```

Comment:
@dirk-thomas hmm I think this is a set comprehension, shouldn't it be `language: extension` instead?

Reply:
Indeed, another fix: 3c76a9e83c9f09ac2580b057718020a8e8313fc5.